src/search/landmarks/landmark_count_heuristic.cc

```diff
@@ -242,25 +245,52 @@ void LandmarkCountHeuristic::generate_preferred_operators(
             FactProxy fact_proxy = effect.get_fact();
             LandmarkNode *lm_node = lgraph->get_node(fact_proxy.get_pair());
             if (lm_node && landmark_is_interesting(
-                    state, reached, *lm_node, all_landmarks_reached)) {
+                state, reached, *lm_node, all_landmarks_reached)) {
```

Comment:
I'm not sure if this change is correct, did/does uncrustify complain here?

Reply:
Indeed, the previous indentation should be restored.